client/src/App.js

```diff
@@ -74,7 +74,16 @@ const App = () => (
           return (
```

Comment:
General question for this file - do we need the 
```
  return ({
    Authorization: token ? `Bearer ${token}` : '',
  });
```
in `getHeaders` for the login/signup to work?

Reply:
Yes right now we do but I'm going to switch us to cookies because of the article you sent me (they're defiantly safer)